Dockerfile

```diff
@@ -0,0 +1,15 @@
+# The jvs image.
+ARG JVS_IMAGE
+
+FROM ${JVS_IMAGE}
```

Comment:
I still recommend directly putting the JVS base image here rather than passing it in with an arg. It's strange using an arg for the base image (I've never seen this style before) and it's easier to find what base image we're using this way.

JVS already releases multi-arch images. Simply use the image without arch, us-docker.pkg.dev/abcxyz-artifacts/docker-images/jvsctl:0.0.5. Per https://docs.docker.com/engine/reference/builder/#from, docker should use the matching arch automatically

Reply:
Good to know. The only reason I am using ARG here is because of the goos. Updated it now to use `us-docker.pkg.dev/abcxyz-artifacts/docker-images/jvsctl:0.0.5` instead